sw/device/silicon_creator/lib/drivers/otbn_unittest.cc

```diff
@@ -227,27 +233,132 @@ TEST_F(DmemReadTest, SuccessWithOffset) {
 
   std::array<uint32_t, 2> test_data = {0};
 
-  EXPECT_EQ(otbn_dmem_read(4, test_data.data(), 2), kErrorOk);
+  otbn_addr_t src_addr = 4;
+  EXPECT_EQ(otbn_dmem_read(2, src_addr, test_data.data()), kErrorOk);
   EXPECT_THAT(test_data, ElementsAre(0x12345678, 0xabcdef01));
 }
 
-class ControlSoftwareErrorsFatalTest : public OtbnTest {};
+class OtbnAppTest : public OtbnTest {};
+
+/*
```

Comment:
nit: why are these test cases commented out? please remove these if the corresponding functions are removed.

Reply:
Oops, that was accidental -- I was trying to figure out which test was failing at one point and missed uncommenting this block. Fixed now!